src/content/en/fundamentals/performance/resource-prioritization.md

```diff
@@ -0,0 +1,351 @@
+“critical path”, most often applied to CSS. This refers to the resources that
+need to be loaded before your initial render, and are thus critical to getting
+the first pixels to the user’s screen.
+
+Previously, the recommendation was to inline this content into your HTML.
+However, in a multi-page, server-side rendered scenario, this quickly grows into
+a lot of wasted bytes. It also makes versioning harder, as any change in the
+critical code invalidates any page that has it inlined.
+
+`<link rel="preload">`, on the other hand, allows you to keep the benefits of
+individual file versioning and caching, while giving you mechanism to request
+the resource as soon as possible.
+
+    <link rel="preload" as="script" href="super-important.js">
+    <link rel="preload" as="style" href="critical.css">
+
+There’s a downside compared to inlining, however: with `<link rel="preload">`
+you’re still subject to an extra roundtrip. This extra roundtrip comes from the
+fact that the browser first has to fetch the HTML, and only then does it find
+out about the next resources.
+
+One way around the extra roundtrip is to use
+[HTTP/2](/web/fundamentals/performance/http2/#server_push)
+push instead, where you preemptively attach the critical assets to the same
+connection through which you’re sending the HTML. This way, you guarantee that
+there’s no downtime between the user’s browser retrieving the HTML and starting
+the download of the critical assets. Be mindful when using HTTP/2 push, though,
+as it’s a very forceful way of controlling the user’s bandwidth usage (“server
+knows best”), and leaves the browser very little room for making its own
+decisions, such as not retrieving a file that is already in its cache!
+
+## Preconnect
+
+`<link rel="preconnect">` informs the browser that your page intends to
+establish a connection to another origin, and that you’d like the process to
+start as soon as possible.
+
+Establishing connections often involves significant time in slow networks,
+particularly where it comes to secure connections, as it may involve DNS
+lookups, redirects, and several round trips to the final server that handles
+the user’s request. Taking care of all this ahead of time can make your
+application feel much snappier to the user without negatively affecting the use
+of bandwidth, as most of the time in establishing a connection is spent waiting,
+rather than exchanging data.
+
+Informing the browser of your intention is as simple as adding a link tag to
+your page:
+
+    <link rel="preconnect" href="https://example.com">
+
+In this case, we’re letting the browser know that we intend to connect to
+`example.com` and retrieve content from there.
+
+Bear in mind that while `<link rel="preconnect">` is pretty cheap, it can still
+take up valuable CPU time, particularly on secure connections. This is
+especially bad if the connection isn’t used within 10 seconds, as the browser
+will close it, wasting all of that early connection work.
```

Comment:
Remove future tense, changing 'will close it' to 'closes it'.

Reply:
I thought that was the last of these. Guess I was wrong! Fixed.